src/NuGet.Clients/VisualStudio/RestoreManager/IVsRestoreGraphItem.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.Runtime.Versioning;
+using NuGet.Versioning;
+
+namespace NuGet.VisualStudio
+{
+    /// <summary>
+    /// Basic interface for restore dependency entity, such as package or assembly
+    /// </summary>
+    [ComImport]
+    [Guid("549e2989-3670-43a1-b8e6-5c2af0446d2f")]
+    public interface IVsRestoreGraphItem
+    {
+        /// <summary>
+        /// Name
+        /// </summary>
+        string Name { get; }
+
+        /// <summary>
+        /// Version range
+        /// </summary>
+        VersionRange Version { get; } 
```

Comment:
You can't use VersionRange directly in ComImport class... It will be better to string instead.